zds/utils/templatetags/emarkdown.py

```diff
@@ -19,11 +19,17 @@
 MAX_ATTEMPTS = 3
 MD_PARSING_ERROR = _('Une erreur est survenue dans la génération de texte Markdown. Veuillez rapporter le bug.')
 
+FORMAT_ENDPOINTS = {
+    'html': '/html',
+    'texfile': '/latex-document',
+    'epub': '/epub',
+    'tex': '/latex',
+}
 
-def _render_markdown_once(md_input, *, to_latex_document=False, to_epub=False, **kwargs):
+
+def _render_markdown_once(md_input, *, format='html', **kwargs):
```

Comment:
`format` est une fonction built-in de python. Il est préférable d'utiliser un autre nom tel que `output_format` or `_format`